bdb/queuedb.c

```diff
@@ -181,7 +181,7 @@ int bdb_queuedb_add(bdb_state_type *bdb_state, tran_type *tran, const void *dta,
             qfnd_seq.epoch = tran->trigger_epoch = comdb2_time_epoch();
         }
 
-        qfnd_seq.seq = 0;
+        qfnd_fnd.seq = 0;
```

Comment:
This looks correct, without this, qfnd_fnd.seq is garbage and line 198 assignment will set qfnd_seq.seq to garbag.